README.md

```diff
@@ -161,30 +161,6 @@ them into utility functions. Original versions of the notebooks can be obtained
 by looking at other notebooks with the same name, or by looking at the history
 of the notebooks.
 
-## JS Testing ##
-
-From the webapp directory
-
-    $ npm install karma --save-dev
-    $ npm install karma-jasmine karma-chrome-launcher --save-dev
-
```

Comment:
I'm glad you took care of this, although in the future, I would prefer to have smaller commits, each of which focuses on one particular action. So I would have put this README into another commit albeit in the same PR.

Reply:
I just removed miniconda.sh - Committed it by mistake